db/schema.rb

```diff
@@ -720,8 +723,11 @@
     t.datetime "created_at", precision: nil, null: false
     t.datetime "updated_at", precision: nil, null: false
     t.boolean "read", default: false
-    t.index ["author_id"], name: "index_warnings_on_author_id"
-    t.index ["community_user_id"], name: "index_warnings_on_community_user_id"
+    t.boolean "is_global", default: false
+    t.bigint "user_id"
+    t.index ["author_id"], name: "index_mod_messages_on_author_id"
+    t.index ["community_user_id"], name: "index_mod_messages_on_community_user_id"
```

Comment:
The index names don't match the table name. Not sure how that happened. Perhaps the table was renamed since the original PR was created?